4-star-nosed_mole/api_definitions/rest/ucs.yaml

```
@@ -99,6 +99,10 @@ paths:
       responses:
         "204":
           description: Multi-part upload successfully updated.
+        "400":
+          description: |
+            The multi-part upload with the given ID can't be
+            set to this status.
```

```suggestion
            Cannot set invalid status for multi-part upload
            with the given ID.
```
Should make it a bit more clear that the status provided is wrong